lib/tasks/wraith.rake

```diff
@@ -0,0 +1,17 @@
+require 'pry'
+require 'rest-client'
+
+desc "check format in wraith."
+task :wraith_format, [:format] do |t, args|
+  format = args[:format]
+  wraith = YAML::load(File.open('test/wraith/config.yaml'))
+  search = RestClient.get "https://www.gov.uk/api/search.json?filter_content_store_document_type=#{format}&count=10"
```

Comment:
I agree, it's confusing. This PR and those linked should be able to give more context: https://github.com/alphagov/govuk-content-schemas/pull/444